### PR TITLE
DNS find zone client fix

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: otcextensions/osclient/dns/v2/zone.py [hunk 1]
```diff
@@ -86,15 +86,28 @@ def get_parser(self, prog_name):
             help=_('UUID or name of the zone.')
         )
 
+        parser.add_argument(
+            '--type',
+            metavar='<type>',
+            choices=['private'],
```
Comment (anton-sidelnikov) [inline otcextensions/osclient/dns/v2/zone.py]: here can be only one choice?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]
question

File: otcextensions/osclient/dns/v2/zone.py (hunk 1 above)
Comment (tischrei) [reply otcextensions/osclient/dns/v2/zone.py]: Yes, this is not an issue. It is related due to the behavior of find and list method in OpenStack and the Huawei adaption.

Comment (tischrei) [reply otcextensions/osclient/dns/v2/zone.py]: Hi Anton, I have changed the choices and description accordingly. Please check again.

**Comment by tischrei** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**